guthrie/roster.json

```diff
@@ -10,6 +10,10 @@
     ]
   },
   "students": [
+    {
+      "name": "Ope Afolabi",
+      "github": "gbopola"
+    },
```

Comment:
You cannot add yourself at the beginning of the list :)

Reply:
oh sorry, I thought I was supposed to that lol.